### DISCUSSION TITLE
Version 0.9.15 has been released

### DISCUSSION THREAD

**Discussion by christianlupus** [nextcloud/cookbook#1199] [Announcements]
Hello everyone,

I just published version 0.9.15 of the cookbook app to the app store.

I am asking everyone to update in a close-time manner as the main reason for the release was a security issue that was found in the code. The risk was relatively low and only covered your recipes. The **rest of your data was never at risk** by the incident. It is nevertheless recommended to update your instances soon.

The release was kept small to prevent any significant regressions but instead push out the fix for the security fast. Though, there have been a few changes in the release that might have an impact:

For now, I **dropped support for NC core versions 19 and 20**. These are marked as past end-of-life now quite some time. If you are still running such old installations, you should really consider upgrading your system!

The security fix forced us to change all API endpoints. The old endpoints are still present but with additional security measures enabled. I informed the authors of any 3rd party apps/scripts/... via the dedicated Matrix channel about the upcoming change. Normally, I suspect they have already implemented in the past a correct way to access the protected resources. However, there might be some regressions if these 3rd party apps were using _creative solutions_. In this case, please do not hesitate to contact us and the authors to get the problem fixed in a timely manner.
I suspect no big problem to appear so far but I want to warn you and keep your eyes open.
The old endpoints have been deprecated (marked as ready to be removed). In the next release, these endpoints will have vanished and the new versioned endpoints must be used instead. **If you are running custom scripts against the API please update them soon.**

Apart from that, a few small bugs were fixed in a quick manner.

I wish you much fun cooking with the app!
Christian

**Comment by meyerchak** [comment]
Thank you!
